sdks/python/apache_beam/transforms/core.py

```diff
@@ -2676,6 +2685,8 @@ def typed(transform):
 
       def inject_default(_, combined):
         if combined:
+          if len(combined) > 1:
+            _LOGGER.error('Cannot inject default values with: %s', combined)
```

Comment:
This error message isn't very actionable (or explanatory). What's going on here is that if the PCollection was non-empty, the combined value is passed through, and otherwise (the else clause below) a default value is provided. 

Maybe "Multiple combined values unexpectedly provided for a global combine" would be clearer?